week-3-practise/level-2/1-map.js

```diff
@@ -12,7 +12,12 @@ var londonLocations = [
   ["Greenwich", "tube", "bus", "river boat"]
 ];
 
-var locationsByBoat;
+var locationsByBoat = londonLocations.map(prova);
```

Comment:
This will end up with undefined values. You should use filter and then map.
```suggestion
var locationsByBoat = londonLocations.filter(location => location.includes("river boat")).map(location => location[0]);
```